### PR TITLE
ZEUS-2083: LNDHub: add custodial label to wallet header

### PR DESCRIPTION
**Pull request by kaloudis**
# Description

Relates to issue: [**ZEUS-2083**](https://github.com/ZeusLN/zeus/issues/2083)

simulator_screenshot_FC0F1209-66A4-4F38-8784-E1464C8B2D7E

simulator_screenshot_D7586E3A-FEB5-47E1-9822-9CB8B88AC497

This pull request is categorized as a:

- [X] New feature
- [ ] Bug fix
- [ ] Code refactor
- [ ] Configuration change
- [ ] Locales update
- [ ] Quality assurance
- [ ] Other

## Checklist
- [ ] I’ve run `yarn run tsc` and made sure my code compiles correctly
- [x] I’ve run `yarn run lint` and made sure my code didn’t contain any problematic patterns
- [x] I’ve run `yarn run prettier` and made sure my code is formatted correctly
- [x] I’ve run `yarn run test` and made sure all of the tests pass

## Testing

If you modified or added a utility file, did you add new unit tests?

- [ ] No, I’m a fool
- [ ] Yes
- [x] N/A

I have tested this PR on the following platforms (please specify OS version and phone model/VM):

- [ ] Android
- [x] iOS

I have tested this PR with the following types of nodes (please specify node version and API version where appropriate):

- [ ] Embedded LND
- [x] LND (REST)
- [ ] LND (Lightning Node Connect)
- [ ] Core Lightning (c-lightning-REST)
- [x] LndHub
- [ ] [DEPRECATED] Core Lightning (Spark)
- [ ] [DEPRECATED] Eclair

### Locales
- [ ] I’ve added new locale text that requires translations
- [ ] I’m aware that new translations should be made on the ZEUS [Transfix page](https://app.transifex.com/ZeusLN/zeus/) and not directly to this repo

### Third Party Dependencies and Packages

- [ ] Contributors will need to run `yarn` after this PR is merged in
- [ ] 3rd party dependencies have been modified:
    * verify that `package.json` and `yarn.lock` have been properly updated
    * verify that dependencies are installed for both iOS and Android platforms

### Other:

- [ ] Changes were made that require an update to the README
- [ ] Changes were made that require an update to onboarding

### CODE REVIEW

**Review by shubhamkmr04 — COMMENTED** [review]
Screenshot 2024-03-26 at 5 28 28 PM

Looks like the badge in android might have some padding issues

File: locales/en.json [hunk 1]
```diff
@@ -1024,7 +1025,11 @@
     "views.Settings.Nostr.relays": "Relays",
     "views.Settings.CurrencyConverter.title": "Currency Converter",
     "views.Settings.CurrencyConverter.enterAmount": "Enter amount",
-    "views.LspExplanation.title": "What are these fees?",
```
Comment (shubhamkmr04) [inline locales/en.json:1027]: I think we removed this locale by mistake

File: views/Settings/CustodialWalletWarning.tsx
Comment (shubhamkmr04) [inline views/Settings/CustodialWalletWarning.tsx:1]: Here on this view, padding and font-size of the content is not matching with other explanatory views like `LspExplanationFees.tsx`

Comment (shubhamkmr04) [reply views/Settings/CustodialWalletWarning.tsx:1]: Screenshot_1711455646
Simulator Screenshot - iPhone 15 Pro - 2024-03-26 at 17 50 40

**Comment by kaloudis** [comment]
New layout, slightly modified.

When the user has a embedded LND wallet already:
simulator_screenshot_08060405-5BE7-47A0-A1B1-2192ACB7BA56

When they don't:
simulator_screenshot_5D573DDF-3EEB-46F8-BA81-8E4B8D2650CF

File: views/Settings/CustodialWalletWarning.tsx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+                        style: {
+                            color: themeColor('text'),
+                            fontFamily: 'PPNeueMontreal-Book'
+                        }
+                    }}
+                    navigation={navigation}
+                />
+                <ScrollView
+                    style={{
+                        flex: 1,
```
Comment (shubhamkmr04) [inline views/Settings/CustodialWalletWarning.tsx:57]: would be good if we add a little padding here

Comment (kaloudis) [reply views/Settings/CustodialWalletWarning.tsx:57]: New margins

Simulator Screenshot - iPhone 8 Plus - 2024-03-27 at 11 55 03

**Comment by shubhamkmr04** [comment]
> Screenshot 2024-03-26 at 5 28 28 PM
> Looks like the badge in android might have some padding issues

Its good to go if we don't wanna focus on this right now.

**Pull request merged**